### DISCUSSION TITLE
Nginx Proxy Manager Helper Script update issue

### DISCUSSION THREAD

**Discussion by yamamototsunemoto** [tteck/Proxmox#3002] [Help Needed]
First a thank you for all your helper scripts. They did make my life much easier. 
I'm running already long time NPM but,I can't run the update in the Nginx proxy Manager. 

When I copy the command in the shell (bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nginxproxymanager.sh)" it trys to update and returns with:  

/ Installing pnpm   
[ERROR] in line 65: exit code 0: while executing command npm install -g pnpm@8.15 &> /dev/null

Just using command update returns with:  -bash: update: command not found

Any idea what could be the issue. I'm too green to undertand what's going on here.

**Comment by JaviSolo** [comment]
Same here, any idea how to solve it?

**Comment by tteck** [comment]
Maybe this will help, https://github.com/tteck/Proxmox/issues/2706#issuecomment-2041056086
